install/v0.7.0/kfp-tekton.yaml

```diff
@@ -0,0 +1,2819 @@
+  name: tekton-pipelineloop-controller-tenant-access
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tekton-pipelineloop-controller-tenant-access
+subjects:
+- kind: ServiceAccount
+  name: tekton-pipelineloop-controller
+  namespace: tekton-pipelines
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipeline-loops
+  name: tekton-pipelineloop-webhook-cluster-access
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tekton-pipelineloop-webhook-cluster-access
+subjects:
+- kind: ServiceAccount
+  name: tekton-pipelineloop-webhook
+  namespace: tekton-pipelines
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipeline-loops
+  name: tekton-pipelineloop-webhook-leaderelection
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tekton-pipelineloop-leader-election
+subjects:
+- kind: ServiceAccount
+  name: tekton-pipelineloop-webhook
+  namespace: tekton-pipelines
+---
+apiVersion: v1
+data:
+  archive_logs: "false"
+  artifact_bucket: mlpipeline
+  artifact_endpoint: minio-service.kubeflow:9000
+  artifact_endpoint_scheme: http://
+  artifact_image: minio/mc:RELEASE.2020-11-25T23-04-07Z
+  artifact_script: |-
+    #!/usr/bin/env sh
+    push_artifact() {
+        tar -cvzf $1.tgz $2
+        mc cp $1.tgz storage/$ARTIFACT_BUCKET/artifacts/$PIPELINERUN/$PIPELINETASK/$1.tgz
+    }
+    push_log() {
+        cat /var/log/containers/$PODNAME*$NAMESPACE*step-main*.log > step-main.log
+        push_artifact main-log step-main.log
+    }
+    strip_eof() {
+        awk 'NF' $2 | head -c -1 > $1_temp_save && cp $1_temp_save $2
+    }
+    mc config host add storage ${ARTIFACT_ENDPOINT_SCHEME}${ARTIFACT_ENDPOINT} $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY
+  inject_default_script: "true"
+  strip_eof: "false"
+  track_artifacts: "true"
+kind: ConfigMap
+metadata:
+  labels:
+    application-crd-id: kubeflow-pipelines
+  name: kfp-tekton-config
+  namespace: kubeflow
+---
+apiVersion: v1
+data:
+  METADATA_GRPC_SERVICE_HOST: metadata-grpc-service
+  METADATA_GRPC_SERVICE_PORT: "8080"
+kind: ConfigMap
+metadata:
+  labels:
+    application-crd-id: kubeflow-pipelines
+    component: metadata-grpc-server
+  name: metadata-grpc-configmap
+  namespace: kubeflow
+---
+apiVersion: v1
+data:
+  viewer-pod-template.json: |-
+    {
+        "spec": {
+            "serviceAccountName": "kubeflow-pipelines-viewer"
+        }
+    }
+kind: ConfigMap
+metadata:
+  labels:
+    application-crd-id: kubeflow-pipelines
+  name: ml-pipeline-ui-configmap
+  namespace: kubeflow
+---
+apiVersion: v1
+data:
+  appName: pipeline
+  appVersion: 1.3.0
```

Comment:
should this be `1.4.0` in anticipation of our [KFP `1.4.0` rebase](https://github.com/kubeflow/kfp-tekton/pull/481)?

Reply:
i can rebase this pr once we merged 1.4.0. right now the base manifest for this PR is 1.3.0